lint/testdata/dummyLen/positive_tests.go

```diff
@@ -0,0 +1,12 @@
+package checker_test
+
+func f() {
+	a := []int{}
+
+	/// useless len comparison len(a) >= 0, always true
```

Comment:
`useless len comparison` part is both redundant and somewhat offensive.
Maybe just `len(a) >= 0 is always true` is enough?
Same goes for other warnings.